components/dashboard/src/data/featureflag-query.ts

```diff
@@ -24,6 +24,7 @@ const featureFlags = {
     enableDedicatedOnboardingFlow: false,
     usageDownload: false,
     phoneVerificationByCall: false,
+    doRetryUserLoader: true,
```

Comment:
@AlexTugarev Would that work?
Sequence:
 - on first retry (on first load of the dashboard with this version), we always go with true 
 - when we fetch another value, we could interrupt/break potential re-try cases with this

Reply:
nice!